src/components/interaction-designer/Block.vue

```diff
@@ -135,7 +136,7 @@
             @mouseleave="exitMouseLeave(exit)">
             <span class="block-exit-tag-text align-self-center"
                   v-b-tooltip.hover.top="exit.test">
-              {{ visibleExitTag(key, exit) }}
+              {{exit.name || '(untitled)'}}
```

Comment:
I think we should localized this `untitled`, if we cannot do it now, let's track it somewhere in a JIRA ticket ;)